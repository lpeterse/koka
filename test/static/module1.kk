public module module1 {
  import static/modA
  import static/modB

  alias ok = aint
}
