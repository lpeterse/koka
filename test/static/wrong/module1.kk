module static/wrong/module1 {
  import static/modA
  import static/modB

  alias ok = int
}
