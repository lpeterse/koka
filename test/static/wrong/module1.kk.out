test/static/wrong/module1.kk(5,14): error: Type int is ambiguous.
 hint: It can refer to either static/modB/int, static/modA/int, or std/core/int